packages/react-utils/src/createRequiredContext.ts

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+
+/**
+ * Create common context-related values from React with a required context.
+ *
+ * @example
+ * ```typescript
+ * const [useFoo, FooConsumer, FooProvider] = createRequiredContext<FooContextValue, 'Foo'>('Foo')
+ * ```
+ */
+
+export default function createRequiredContext<TContextProps, TName extends string>(
```

Comment:
todo: As this package `@noaignite/react-utils` is a new package we have already adopted the "no default exports" pattern. Should be a named export.

todo: The generic `TName` does not seem to be used anywhere, no need for it to be a generic with the current implementation.